standards_lab/utils/project.py

```diff
@@ -60,13 +63,25 @@ def delete_project(project_name):
 
     if cache.has_key(project_name):
         cache.delete(project_name)
-    if os.path.exists(path):
-        try:
-            os.rmdir(path)
-        except OSError:
-            files = os.listdir(path)
+
+    # We don't use shutil.rmtree() because
+    # when we do https://github.com/OpenDataServices/standards-lab/issues/143 the Django file API has no equivalent,
+    # so we may as well set up the idea of looking for files and deleting individually now
```

Comment:
If you _really_ think this is jarring and want a comment on justifying out API choices, I'd be more general, like "We implement our own recursive delete for future compatibility with django file storage API. See https://github.com/OpenDataServices/standards-lab/issues/143 " ... Also technically not all storage backends emulate a filesystem anyway...

Reply:
Jarring's not the word I'd use, it's more my first thought on seeing this was "isn't there a recursive option that would work?" and to save any future coders time wondering the same thing (or maybe even changing it when we are about to change it back), it seems good to comment it. I'd prefer my comment as it's clearer why the comment is there, and it's clearer why we don't use a recursive function.

Reply:
Also as said above this comment probably won't last long :-)